Dockerfile

```diff
@@ -101,6 +101,8 @@ RUN . $UNDERLAY_WS/install/setup.sh && \
       --from-paths src \
       --skip-keys " \
         slam_toolbox \
+        turtlebot3_gazebo \
+        gazebo_ros_pkgs \
```

Comment:
Shouldn't you add these here too?

https://github.com/open-navigation/navigation2/blob/d0a65d5afbab3b758408b79ebf5cd344b3377b00/.circleci/config.yml#L110-L112